### ISSUE TITLE
can I have the training config for Qwen 0.5B and 1.5B

### ISSUE THREAD

**Issue by vengdeng** [hkust-nlp/simpleRL-reason#61]
Dear authors, thank you for your contribution, can I have the the training config for Qwen 0.5B and 1.5B

Thank you in advance!

**Comment by rasdani** [comment]
pretty much all of it is in the code and in the paper.
I reproduced successfully on two 4090 with this implementation https://github.com/rasdani/smolR1 (some tweaks due to limited compute though).

**Comment by Zeng-WH** [comment]
Thank you for your attention. The training config could remain consistent with our code, and for the data, please select:  

- **0.5B**: [https://huggingface.co/datasets/hkust-nlp/SimpleRL-Zoo-Data/tree/main/simplelr_abel_level1to4](https://huggingface.co/datasets/hkust-nlp/SimpleRL-Zoo-Data/tree/main/simplelr_abel_level1to4)  
- **1.5B**: [https://huggingface.co/datasets/hkust-nlp/SimpleRL-Zoo-Data/tree/main/simplelr_abel_level3to5](https://huggingface.co/datasets/hkust-nlp/SimpleRL-Zoo-Data/tree/main/simplelr_abel_level3to5)

**Comment by aidando73** [comment]
@vengdeng here were my training scripts for Qwen 0.5B: https://github.com/aidando73/simpleRL-reason/pull/2

You can view my wandb metrics here: https://wandb.ai/aidando73-personal/simplerl-math-grpo-single-node?nw=nwuseraidando73

**Comment by ericg108** [comment]
I am curious about the config too. I am training on a single A800 GPU, but after a few steps I will get CUDA OOM error.. wondered how to modify the config properly

**Comment by rasdani** [comment]
reduce per device batch size, use gradient accumulation or shorten completions, if necessary

**Comment by vengdeng** [comment]
> Thank you for your attention. The training config could remain consistent with our code, and for the data, please select:
> 
> * **0.5B**: https://huggingface.co/datasets/hkust-nlp/SimpleRL-Zoo-Data/tree/main/simplelr_abel_level1to4
> * **1.5B**: https://huggingface.co/datasets/hkust-nlp/SimpleRL-Zoo-Data/tree/main/simplelr_abel_level3to5

Thank you!

**Comment by vengdeng** [comment]
> [@vengdeng](https://github.com/vengdeng) here were my training scripts for Qwen 0.5B: [aidando73#2](https://github.com/aidando73/simpleRL-reason/pull/2)
> 
> You can view my wandb metrics here: https://wandb.ai/aidando73-personal/simplerl-math-grpo-single-node?nw=nwuseraidando73

thanks!

**Comment by vengdeng** [comment]
> Thank you for your attention. The training config could remain consistent with our code, and for the data, please select:
> 
> * **0.5B**: https://huggingface.co/datasets/hkust-nlp/SimpleRL-Zoo-Data/tree/main/simplelr_abel_level1to4
> * **1.5B**: https://huggingface.co/datasets/hkust-nlp/SimpleRL-Zoo-Data/tree/main/simplelr_abel_level3to5

one more question , i found in paper you said you use Medium for Qwen- 2.5-0.5B?

**Comment by HYZ17** [comment]
Yes, Medium means MATH lv.1–4 in our paper, which corresponds to this link https://huggingface.co/datasets/hkust-nlp/SimpleRL-Zoo-Data/tree/main/simplelr_abel_level1to4 .

**Comment by HYZ17** [comment]
Yes, Medium means MATH lv.1–4 in our paper, which corresponds to this link https://huggingface.co/datasets/hkust-nlp/SimpleRL-Zoo-Data/tree/main/simplelr_abel_level1to4 .

**Comment by Joe-2002** [comment]
Hello, thank you for your work, I would like to know whether your work is based on Instruct model or base model.
If we use Instruct Model, can we solve the problem of poor command compliance for 0.5B or 1.5B model mentioned in the paper?
@Zeng-WH @HYZ17

**Comment by Zeng-WH** [comment]
Thank you for your attention. Yes, you can do some SFT or use Instruct model to help the model follow instructions better. However, as we mentioned in our paper (and also noted in recent LLaMA 4), try to keep SFT lightweight, otherwise it may restrict the model's exploration capabilities.

**Comment by HYZ17** [comment]
> Hello, thank you for your work, I would like to know whether your work is based on Instruct model or base model. If we use Instruct Model, can we solve the problem of poor command compliance for 0.5B or 1.5B model mentioned in the paper? [@Zeng-WH](https://github.com/Zeng-WH) [@HYZ17](https://github.com/HYZ17)

In our paper, we conduct RL on base model (instead of the instruct model) and we find that model can follow the instruction during training. So you may start from the base model first. Thank you ~

**Comment by xueyongfu11** [comment]
It is difficult for me to reproduce the gradual increase in response length during the training of the qwen2.5 - 0.5B model as described in the paper. The several wandb metrics mentioned above also show a gradual decline in response length with training, and my training also sees a gradual decrease in response length.
Someone can help me？

![Image](https://github.com/user-attachments/assets/c1837584-b09c-432e-a167-2ba860d3b906)

**Comment by oncleJules** [comment]
> [@vengdeng](https://github.com/vengdeng) here were my training scripts for Qwen 0.5B: [aidando73#2](https://github.com/aidando73/simpleRL-reason/pull/2)
> 
> You can view my wandb metrics here: https://wandb.ai/aidando73-personal/simplerl-math-grpo-single-node?nw=nwuseraidando73

hey my friend, i clicked the link but did not see a full script for training

could you further indicate where to find it?
